cmd/ocitools/validate.go

```diff
@@ -381,6 +386,47 @@ func checkLinux(spec rspec.Spec, rootfs string, hostCheck bool) (msgs []string)
 	return
 }
 
+func checkLinuxResources(r rspec.Resources, hostCheck bool) (msgs []string) {
+	logrus.Debugf("check linux resources")
+
+	if r.OOMScoreAdj != nil && (int(*r.OOMScoreAdj) < -1000 || int(*r.OOMScoreAdj) > 1000) {
+		msgs = append(msgs, fmt.Sprintf("OOMScoreAdj range is from -1000 to 1000"))
```

Comment:
It looks like a good advice, we can implement it later.
Before implementing that,  I think keep min and max check is OK.
Maybe they are some short of thing, but Ocitools and runtime schema are two projects and ocitools does not have these yet.